src/components/Form/CreateProfileForm/ProfileData/SalesChannels.tsx

```diff
@@ -135,7 +135,14 @@ const SalesChannels = ({ entityType }: Props) => {
                     title={
                       index + 1 >= 2 ? `Indirizzo ${index + 1}` : `Indirizzo`
                     }
-                    description="Inserisci l'indirizzo completo del/i punto/i vendita per permetterne la corretta visualizzazione del pin sulla mappa in app IO"
+                    description={(() => {
+                      switch (entityType) {
+                        case EntityType.Private:
+                          return "Inserisci l'indirizzo completo della sede. Servirà agli utenti per usufruire delle agevolazioni del tuo operatore, se richiedono una sede fisica.";
+                        case EntityType.PublicAdministration:
+                          return "Inserisci l'indirizzo completo della sede. Servirà agli utenti per usufruire delle opportunità del tuo operatore, se richiedono una sede fisica.";
+                      }
+                    })()}
```

Comment:
To improve code readability may we isolate the function outside of the component?

Reply:
How do you think readability would improve?
In my opinion the code inderection created is not worth since conditional rendering is a standard JSX react technique.

Reply:
> In my opinion the code indirection created is not worth since conditional rendering is a standard JSX react technique.

I can understand and totally agree with this point!

In this specific case we're talking about a component code made of ~300 lines of code, with nested functions to render Fields and FieldArray, I guess would be better to move away from the rendering function just for code organization. 

In addition starting from this PR the automatic lint check on the CI is raising an increase of cognitive-complexity on the component

<img width="960" alt="image" src="https://github.com/pagopa/cgn-onboarding-portal-frontend/assets/3959405/11808fad-29c3-408e-9380-4f28f436447a">

here the result on my terminal. Just splitting the code function would be enough at the moment, do you have any better idea?

Reply:
Almost all the cognitive complexity in that component comes from conditional rendering.
Also personally i do not count lines of code if they are only markup or style, trying to follow the the contents of a component where markup is extracted into separate components only beacause of LOC (constantly jumping to different lines or files).
Guidelines and best practices that i found to be more useful is to extract a component only in 2 cases:
- it is used in more than 3 places, with low parametrization
- it needs to be memoized (mostli for list/grid items if profiler suggest so)

Mu suggestion is to remove JSX from cognitive complexity count all together (so only hooks are considered)

Reply:
> Mu suggestion is to remove JSX from cognitive complexity count all together (so only hooks are considered)
Great 💪🏻 , can we address in this PR?